vault/resource_kubernetes_secret_backend_role.go

```diff
@@ -70,6 +71,11 @@ func kubernetesSecretBackendRoleResource() *schema.Resource {
 			},
 			fieldAllowedKubernetesNamespaceSelector: {
 				Type: schema.TypeString,
+				// We rebuild the attached JSON string to a simple singleline
+				// string. This makes terraform not want to change when an extra
+				// space is included in the JSON string.
+				StateFunc:    NormalizeDataJSONFunc(name),
+				ValidateFunc: ValidateDataJSONFunc(name),
```

Comment:
I am not actually sure if we want this because this field accepts yaml as well. We don't normalize the input for `generated_role_rules` in this resource. 🤔

Reply:
But if we don't normalize it somehow we can get drift.